mlflow/utils/validation.py

```diff
@@ -60,7 +60,7 @@
 MAX_EXPERIMENTS_LISTED_PER_PAGE = 50000
 MAX_DATASET_NAME_SIZE = 500
 MAX_DATASET_DIGEST_SIZE = 36
-MAX_DATASET_SCHEMA_SIZE = 65535  # 64KB -1 (the db limit for TEXT column)
+MAX_DATASET_SCHEMA_SIZE = 16777215  # 16MB -1 (the db limit for MEDIUMTEXT column)
```

Comment:
Q: Do we have so long schema string ?

Reply:
Yes, databricks backend has supported it https://src.dev.databricks.com/databricks/universe@7860d65beb6559a9f7c4b4b68476174cab175ace/-/blob/mlflow/db/tracking/migrations/V46__datasets_change_schema_to_medium_text.jsonnet